### PR TITLE
Cleanup setup.py

### PR DESCRIPTION
**Pull request by SShrike**
I went through and made the string quotes consistent, added some classifiers, and updated the authors field to say 'The Matrix Developers (matrix.org)', which I thought looked nicer than just 'matrix.org'.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,24 +24,27 @@ def exec_file(names):
...
-    description="Client-Server SDK for Matrix",
-    long_description=read_file(("README.rst",)),
-    author="matrix.org",
-    author_email="team@matrix.org",
-    url="https://github.com/matrix-org/matrix-python-sdk",
+    name='matrix_client',
+    version=exec_file(('matrix_client', '__init__.py',))['__version__'],
+    description='Client-Server SDK for Matrix',
+    long_description=read_file(('README.rst')),
+    author='The Matrix Developers (matrix.org)',
```
Comment (Half-Shot) [inline setup.py]: Only thing I want to add is could you make it 'Matrix.org team' as:
* Developers is a little vague, and technically speaking the SDK is still a core matrix project.
* The site refers to the team as 'Matrix.org team' so I'd like to keep it consistent.

**Comment by Half-Shot** [comment]
LGTM.

@erikjohnston ?

File: setup.py (hunk 1 above)
Comment (SShrike) [inline setup.py]: I see, I wrote "The Matrix Developers" as it seems to be a common author name for projects with many contributors (see Rust for example). I'll go ahead and change it though if you'd like.

**Comment by erikjohnston** [comment]
@matrixbot ok to test

**Comment by SShrike** [comment]
*Sigh*, did I break something?

**Comment by SShrike** [comment]
Oh god what have I done?

```
Traceback (most recent call last):
  File "setup.py", line 30, in <module>
    long_description=read_file(('README.rst')),
  File "setup.py", line 12, in read_file
    with codecs.open(file_path, encoding=encoding) as f:
  File "/usr/lib/python2.7/codecs.py", line 881, in open
    file = __builtin__.open(filename, mode, buffering)
IOError: [Errno 2] No such file or directory: '/var/lib/jenkins/jobs/PythonSDK-PR/workspace/R/E/A/D/M/E/./r/s/t'
```

**Comment by Half-Shot** [comment]
:D Hah

**Comment by Half-Shot** [comment]
Alright. Merging.

**Comment by SShrike** [comment]
:tada:

**Pull request merged**